app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,20 @@
+module OrdersHelper
+  def cal_total_order order
+    @total = 0
+    order_details = order.order_details
+    order_details.each do |order_detail|
```

Comment:
tìm lại 1 method khác, vừa duyệt và tính toán trong collection, trả về kết quả tính toán  
-> có method này nhé, tìm key "ruby collection inject`

Reply:
Dạ. E sài inject thì rubocop nó bảo nên sài reduce nên e sài reduce ạ